File: paywellservices/src/main/java/com/cloudwell/paywell/services/activity/eticket/airticket/source/model/Airport.java

package com.cloudwell.paywell.services.activity.eticket.airticket.source.model;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class Airport {

    @SerializedName("airport_name")
    private String mAirportName;
    @SerializedName("city")
    private String mCity;
    @SerializedName("country")
    private String mCountry;
    @SerializedName("iata")
    private String mIata;
    @SerializedName("icao")
    private String mIcao;
    @SerializedName("id")
    private String mId;
    @SerializedName("iso")
    private String mIso;
    @SerializedName("state")
    private String mState;
    @SerializedName("time_zone")
    private String mTimeZone;

    public String getAirportName() {
        return mAirportName;
    }

    public void setAirportName(String airportName) {
        mAirportName = airportName;
    }

    public String getCity() {
        return mCity;
    }

    public void setCity(String city) {
        mCity = city;
    }

    public String getCountry() {
        return mCountry;
    }

    public void setCountry(String country) {
        mCountry = country;
    }

    public String getIata() {
        return mIata;
    }

    public void setIata(String iata) {
        mIata = iata;
    }

    public String getIcao() {
        return mIcao;
    }

    public void setIcao(String icao) {
        mIcao = icao;
    }

    public String getId() {
        return mId;
    }

    public void setId(String id) {
        mId = id;
    }

    public String getIso() {
        return mIso;
    }

    public void setIso(String iso) {
        mIso = iso;
    }

    public String getState() {
        return mState;
    }

    public void setState(String state) {
        mState = state;
    }

    public String getTimeZone() {
        return mTimeZone;
    }

    public void setTimeZone(String timeZone) {
        mTimeZone = timeZone;
    }

}

File: paywellservices/src/main/java/com/cloudwell/paywell/services/activity/eticket/airticket/serach/citySerach/model/Airport.kt
package com.cloudwell.paywell.services.activity.eticket.airticket.serach.citySerach.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize


@Parcelize
@Entity
class Airport : Parcelable {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    var id: String = ""

    @ColumnInfo(name = "airport_name")
    @SerializedName("airport_name")
    var airportName: String = ""

    @ColumnInfo(name = "city")
    @SerializedName("city")
    var city: String = ""

    @ColumnInfo(name = "country")
    @SerializedName("country")
    var country: String = ""

    @ColumnInfo(name = "iata")
    @SerializedName("iata")
    var iata: String = ""


    @ColumnInfo(name = "icao")
    @SerializedName("icao")
    var icao: String = ""


    @ColumnInfo(name = "iso")
    @SerializedName("iso")
    var iso: String = ""


    @ColumnInfo(name = "state")
    @SerializedName("state")
    var state: String = ""

    @ColumnInfo(name = "time_zone")
    @SerializedName("time_zone")
    var timeZone: String = ""


    @ColumnInfo(name = "status")
    @SerializedName("status")
    var status: String = ""


}
